browser-interface/packages/shared/meta/sagas.ts

```diff
@@ -64,19 +64,33 @@ async function fetchFeatureFlagsAndVariants(network: ETHEREUM_NETWORK): Promise<
     const flags = new URLSearchParams(location.search)
     flags.forEach((_, key) => {
       if (key.startsWith(`DISABLE_`)) {
-        const featureName = key.replace('DISABLE_', '').toLowerCase() as FeatureFlagsName
-        if (featureName in flagsAndVariants.variants) {
-          flagsAndVariants.flags[featureName] = true
-          flagsAndVariants.variants[featureName].enabled = false
+        const nameVariant = key.replace('DISABLE_', '').toLowerCase().split(':')
+        const name = nameVariant[0] as FeatureFlagsName
+        if (name in flagsAndVariants.variants) {
+          flagsAndVariants.flags[name] = true
+          flagsAndVariants.variants[name].enabled = false
         } else {
-          flagsAndVariants.flags[featureName] = false
+          flagsAndVariants.flags[name] = false
         }
       } else if (key.startsWith(`ENABLE_`)) {
-        const featureName = key.replace('ENABLE_', '').toLowerCase() as FeatureFlagsName
-        flagsAndVariants.flags[featureName] = true
-        if (featureName in flagsAndVariants.variants) {
-          flagsAndVariants.variants[featureName].enabled = true
+        const nameVariant = key.replace('ENABLE_', '').toLowerCase().split(':')
+        const name = nameVariant[0] as FeatureFlagsName
+        const variant = nameVariant.length > 1 ? nameVariant[1] : null
+
+        flagsAndVariants.flags[name] = true
+        if (name in flagsAndVariants.variants) {
+          flagsAndVariants.variants[name].enabled = true
         } else {
+          if (variant !== null) {
+            flagsAndVariants.variants[name] = {
+              enabled: true,
+              name: variant,
+              payload: {
+                type: 'string',
+                value: variant
```

Comment:
`name` and `payload.value` are things that can be different.

Although, we are not using `payload.value` this can come back later to bite us. It would be better if we ignore the payload entirely or we use the value of the URL params to set it up;

See: https://github.com/decentraland/unity-renderer/blob/fa8dfb2bd763eea332280b67caf170724257b3d1/browser-interface/packages/shared/meta/selectors.ts#L74-L91

Reply:
Done!